doc/nrf/gs_installing.rst

```diff
@@ -695,3 +564,152 @@ Define the required environment variables as follows, depending on your operatin
+Checking out a branch or tag in the ``nrf`` repository gives you a different version of the manifest file.
+Running ``west update`` will then update the project repositories to the state specified in this manifest file.
+
+.. include:: gs_installing.rst
+   :start-after: west-error-start
+   :end-before: west-error-end
+
+For example, to switch to release |release| of the |NCS|, enter the following commands in the ``ncs/nrf`` directory:
+
+.. parsed-literal::
+   :class: highlight
+
+   git fetch origin
+   git checkout |release|
+   west update
+
+To update to a particular revision (SHA), make sure that you have that particular revision locally before you check it out (by running ``git fetch origin``)::
+
+   git fetch origin
+   git checkout 224bee9055d986fe2677149b8cbda0ff10650a6e
+   west update
+
+To switch to the latest state of development, enter the following commands::
+
+   git fetch origin
+   git checkout origin/master
+   west update
+
+.. note::
+   Run ``west update`` every time you change or modify the current working branch (for example, when you pull, rebase, or check out a different branch).
+   This will bring the project repositories to the matching revision defined by the manifest file.
+
+.. _repo_move:
+
+Pointing the repositories to the right remotes after they were moved
+====================================================================
+
+Before |NCS| version 1.3.0, the Git repositories were moved from the NordicPlayground GitHub organization to the nrfconnect organization.
+They were also renamed, replacing the ``fw-nrfconnect-`` prefix with ``sdk-``.
+
+If you cloned the repositories before the move, your local repositories and forks of the |NCS| repositories are automatically be redirected to the new ones.
+However, you should point them directly to their new locations as described in this section.
+
+The full list of repositories with their old and new URLs can be found in the following table:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Old URL
+     - New URL
+     - File system path
+
+   * - https:\ //github.com/NordicPlayground/fw-nrfconnect-nrf
+     - https://github.com/nrfconnect/sdk-nrf
+     - :file:`ncs/nrf`
+
+   * - https:\ //github.com/NordicPlayground/fw-nrfconnect-zephyr
+     - https://github.com/nrfconnect/sdk-zephyr
+     - :file:`ncs/zephyr`
+
+   * - https:\ //github.com/NordicPlayground/fw-nrfconnect-mcuboot
+     - https://github.com/nrfconnect/sdk-mcuboot
+     - :file:`ncs/bootloader/mcuboot`
+
+   * - https:\ //github.com/NordicPlayground/fw-nrfconnect-mcumgr
+     - https://github.com/nrfconnect/sdk-mcumgr
+     - :file:`ncs/modules/lib/mcumgr`
+
+   * - https:\ //github.com/NordicPlayground/nrfxlib
+     - https://github.com/nrfconnect/sdk-nrfxlib
+     - :file:`ncs/nrfxlib`
+
+
+Before you change the remotes, rename any personal forks that you have of the
+|NCS| repositories to their new names.
+
+To do so, visit your personal fork in a browser and edit the name there.
```

Comment:
Numbered steps perhaps? I mean 1. edit the remote name in browser and 2. rename the actual remotes

Reply:
Updated.